.github/workflows/sync_from_airtable.yml

```diff
@@ -0,0 +1,330 @@
+                                            owner: OWNER,
+                                            repo: REPO,
+                                            name: label
+                                        });
+                                        core.info(`Label created: ${label}`)
+                                        await github.rest.issues.addLabels({
+                                            owner: OWNER,
+                                            repo: REPO,
+                                            issue_number: NUMBER,
+                                            labels: label
+                                        });
+                                        core.info(`Label added: ${label}`)
+                                    } catch (error) {
+                                        console.error(`Label not added: ${error}`)
+                                    }
+                                }
+                            }))
+                        }
+                    }
+                }
+                if (removeLabels && removeLabels.length > 0) {
+                    try {
+                        await Promise.all(removeLabels.map(async (name) => {
+                            await github.rest.issues.removeLabel({
+                                owner: OWNER,
+                                repo: REPO,
+                                issue_number: NUMBER,
+                                name
+                            });
+                            core.info(`Remove Label: ${name}`)
+                        }));
+                    } catch (err) {
+                        if (!error.includes('Label does not exist')) {
+                          core.error(`Label not removed: ${err}`)
+                        }   
+                    }
+                }
+            }
+
+  milestoned_demilestoned:
+    name: Milestoned or Demilestoned
+    needs: existing_issue
+    if: (fromJSON(needs.existing_issue.outputs.github_issue).status == 200 ) && (github.event.client_payload.action == 'milestoned/demilestoned')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6
+        env:
+          DATA: ${{ fromJSON(env.DATA) }}
+          NUMBER: ${{env.NUMBER}}
+          OWNER: ${{env.OWNER}}
+          REPO: ${{env.REPO}}
+        with:
+          script: |
+            const { DATA, NUMBER, OWNER, REPO } = process.env;
+            const { milestone } = JSON.parse(DATA);
+            const repoName = `${OWNER}/${REPO}`;
+            try {
+                if (milestone) {
+                    const { repo, milestone_number } = milestone;
+                    if (repoName === repo) {
+                        await github.rest.issues.update({
+                            owner: OWNER,
+                            repo: REPO,
+                            issue_number: NUMBER,
+                            milestone: milestone_number || null,
```

Comment:
Is it possible for a milestone to exist in airtable but not github? Do we know what happens when trying to set a milestone that doesn't exist? (This is assuming `env.DATA` is airtable info that you converted to github's schema)